docs/_themes/searxng/static/searxng.css

```diff
@@ -20,6 +20,10 @@ div.sidebar {
   border-radius: 3pt;
 }
 
+div.sphinxsidebar p.caption {
+ display: none;
```

Comment:
Most of the file uses 2 spaces for indentation.

Reply:
Oops .. fixed / thanks!